gst/nnstreamer/tensor_common.h

```diff
@@ -85,20 +85,18 @@ G_BEGIN_DECLS
     GST_TENSOR_TEXT_CAPS_STR "; " \
     GST_TENSOR_OCTET_CAPS_STR
 
-/** @todo I'm not sure if the range is to be 1, 65535 or larger */
-#define GST_TENSOR_DIM_RANGE "(int) [ 1, 65535 ]"
-#define GST_TENSOR_RATE_RANGE "(fraction) [ 0/1, 2147483647/1 ]"
+#define GST_TENSOR_RATE_RANGE "(fraction) [ 0, max ]"
```

Comment:
What is the difference with "#define GST_TENSOR_RATE_RANGE "(fraction) [ 0/1, 2147483647/1 ]""

Reply:
the value is not different, indeed. (0 ~ 2147483647)